src/components/Slider/Slider.spec.tsx

```diff
@@ -0,0 +1,32 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { mount } from "@cypress/react";
+import { useState } from "react";
+import Slider from "./Slider";
+import css from "./Slider.module.css";
+
+const Slider_Items = ["abc", "def", "ghi"];
```

Comment:
not sure if this is a convention of frontify (@SamuelAlev) but @NadiaPosch IMO either write everything in capital letters eg `SLIDER_ITEMS` or classical snake_case. this is more of a gusto thingy but personally i never saw this kind of variable "convention" 😉